portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,131 @@
+    </head>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+        <h1>Cynthia HU</h1>
+        <ul class="navbar">
+            <li><a href="#About">About</a></li>
+            <li><a href="#TechCorner">Tech Corner</a></li>
+            <li><a href="#TravelLog">Travel Log</a></li>
+            <li><a href="#Contact">Contact</a></li>
+        </ul>
+        <div id="About">
+            <h2>About</h2>
+            <img src="/images/profile3.jpg" alt="me in Hokkaido, Japan" 
+                class="image"
+                width="400" height="300"
+                srcset="/images/profile3.jpg 400w, 
+                        /images/profile3.jpg 800w, 
+                        /images/profile3.jpg 1200w"
+                sizes="(max-width: 400px) 200px,
+                        (max-width: 800px) 400px,
+                        (max-width: 1200px) 800px,
+                        1200px"
+            >
+            <p class="intro">Hello there! This is Cynthia Hu and welcome to my portfolio.</p>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.cmu.edu/">Where I am half way through my journey to a computer science major and psychology minor...</a><br>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.google.com/maps/place/Shenzhen,+Guangdong+Province,+China/@22.5549176,113.7736861,10z/data=!3m1!4b1!4m5!3m4!1s0x3403f408d0e15291:0xfdee550db79280c9!8m2!3d22.543096!4d114.057865">Where I spent the first 12 years of my life...</a><br>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.google.com/maps/place/Fremont,+CA/data=!4m2!3m1!1s0x808fbf46b7e8caf7:0x8ada313b89d888d4?sa=X&ved=2ahUKEwj4sOqN_NfpAhWX4J4KHU3xDUMQ8gEwIHoECBoQBg">Where I've been spending my life at after the first 12 years...</a><br>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.allure.com/story/aquarius-zodiac-sign-personality-traits">If you are into astrology...</a><br>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.16personalities.com/esfj-personality">If you are into 16 personalities...</a><br>
+            <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/watch?v=tWN517PBkGc">Oh, and I dance! Here is a throwback to good old days in highschool...</a><br>
+            
+        </div>
+
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+
+        <div id="greeting-container"></div>
+
+        <div id="TechCorner">
+            <h2>Tech Corner</h2>
+        </div>
+
+        <div id="TravelLog">
+            <h2>Travel Log</h2>
+            <p>Oh the places I go...</p>
+            <!-- Slideshow container -->
+            <div class="slideshow-container"> 
```

Comment:
Great implementation of a flipping photo carousel.

One minor improvement suggested (optional to work on based on your preference): The photos are the last section of the home page and they are not the same height. Therefore while flipping the photos, sometimes the viewport scroll position was forced upwards because of a photo with less height. For good viewing experience the scroll position should remain the same unless user scrolls the page.

You could fix it by simply setting a fixed height for this section.

Reply:
instead of fixing the height, I increased the vertical padding of the buttons so that the user doesn't have to move the mouse to still click the button.

Reply:
Smart solution.